Tests/SwiftLintFrameworkTests/CommandTests.swift

```diff
@@ -280,93 +285,99 @@ class CommandTests: XCTestCase {
     func testSuperfluousDisableCommandsIgnoreDelimiter() {
         let longComment = "Comment with a large number of words that shouldn't register as superfluous"
         XCTAssertEqual(
-            violations(Example("// swiftlint:disable nesting - \(longComment)\nprint(123)\n"))[0]
+            TestHelpers.violations(Example("// swiftlint:disable nesting - \(longComment)\nprint(123)\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
         XCTAssertEqual(
-            violations(Example("// swiftlint:disable:next nesting - Comment\nprint(123)\n"))[0]
+            TestHelpers.violations(Example("// swiftlint:disable:next nesting - Comment\nprint(123)\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
         XCTAssertEqual(
-            violations(Example("print(123) // swiftlint:disable:this nesting - Comment\n"))[0]
+            TestHelpers.violations(Example("print(123) // swiftlint:disable:this nesting - Comment\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
         XCTAssertEqual(
-            violations(Example("print(123)\n// swiftlint:disable:previous nesting - Comment\n"))[0]
+            TestHelpers.violations(Example("print(123)\n// swiftlint:disable:previous nesting - Comment\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
     }
 
     func testInvalidDisableCommands() {
         XCTAssertEqual(
-            violations(Example("// swiftlint:disable nesting_foo\nprint(123)\n"))[0]
+            TestHelpers.violations(Example("// swiftlint:disable nesting_foo\nprint(123)\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
         XCTAssertEqual(
-            violations(Example("// swiftlint:disable:next nesting_foo\nprint(123)\n"))[0]
+            TestHelpers.violations(Example("// swiftlint:disable:next nesting_foo\nprint(123)\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
         XCTAssertEqual(
-            violations(Example("print(123) // swiftlint:disable:this nesting_foo\n"))[0]
+            TestHelpers.violations(Example("print(123) // swiftlint:disable:this nesting_foo\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
         XCTAssertEqual(
-            violations(Example("print(123)\n// swiftlint:disable:previous nesting_foo\n"))[0]
+            TestHelpers.violations(Example("print(123)\n// swiftlint:disable:previous nesting_foo\n"))[0]
                 .ruleIdentifier,
             "superfluous_disable_command"
         )
 
         XCTAssertEqual(
-            violations(Example("print(123)\n// swiftlint:disable:previous nesting_foo \n")).count,
+            TestHelpers.violations(Example("print(123)\n// swiftlint:disable:previous nesting_foo \n")).count,
```

Comment:
Is there a need to put everything under a static function?
It's not that this should be included in PR

Reply:
Not needed, but I thought it made it clearer where things were coming from. But I'm happy to move it to a global, since it's still nicely scoped inside the test helpers, so we can use `SwiftLintFrameworkTests.violations` where we need to disambiguate.

Side note: I'd prefer `violations(in:...)` instead of `violations(...)`, but don't want to change too much at once.